signac/common/errors.py

```diff
@@ -22,9 +22,5 @@ class ExportError(Error, RuntimeError):
     pass
 
 
-class FileNotFoundError(Error, FileNotFoundError):
```

Comment:
This seemed like an unused compatibility layer for Python 2 (which did not have `FileNotFoundError`). I removed it.